notebooks/official/workbench/spark/spark_sample_notebook.ipynb

```diff
@@ -0,0 +1,1200 @@
+{
```

Comment:
<p>A few questions about this</p><ul><li>Is there a particular reason why you chose the metadata display to do the validation? Is there something of note users should be looking for? If so, tell them</li><li>Is this only to accomplish that you can successfully connect? If so, if we leave this step out, will the notebook fail in a helpful way later? (showing a 401 or a 403 that indicates the error has to do with cloud storage?) Or will the error be swallowed by another component? If it will fail in a helpful manner later, this probably isn't necessary</li></ul><p> </p><p>I'm kind of bikeshedding here and I def don't want to die on this hill but I am curious</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/720/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>When the user execute to create a bucket successfully, it prints <code>Creating gs://BUCKET_NAME/...</code> and if it fails, the shell shows the appropriate error message. Since it doesn't tell users the message of success, I think it needs a validation step.</p><p> </p><p>In the template notebook, the instruction to validate whether the bucket is created was <code>! gsutil ls -al $BUCKET_URI</code>. However, it prints nothing if the creation was successful as it has no contents, and that's why I choose to print metadata instead.</p><p> </p>